modules/java/misc/mbt/gen_dict.json

```diff
@@ -0,0 +1,170 @@
+{
+  "module_imports": [
+    "java.lang.String"
+  ],
+  "class_ignore_list": [
+    "vpMbDepthNormalTracker",
+    "vpMbKltTracker",
+    "vpMbScanLine",
+    "vpMbTracker",
+    "vpMbtDistanceCylinder",
+    "vpMbtMeEllipse"
+  ],
+  "ManualFuncs": {
+    "VpMbGenericTracker": {
```

Comment:
Not able to properly bind `vpMbGenericTracker(vector_int trackerTypes)`.

Test to add all the variants in `gen_dict.json`?

```
,
      "vpMbGenericTracker" : {
        "j_code" : [
          "//",
          "// manual port",
          "// C++: vpMbGenericTracker()",
          "//",
          "//javadoc: VpMbGenericTracker::vpMbGenericTracker(const std::vector<int> &trackerTypes)\n",
          "public void vpMbGenericTracker(int[] trackerTypes)",
          "{",
          "   vpMbGenericTracker(nativeObj, trackerTypes);",
          "}"
        ],
        "jn_code" : [
          "// C++: vpMbGenericTracker(const std::vector<int> &trackerTypes)",
          "private static native void vpMbGenericTracker(long address, int[] trackerTypes);"
        ],
        "cpp_code" : [
          "//",
          "// manual port",
          "// vpMbGenericTracker(const std::vector<int> &trackerTypes)",
          "//\n",
          "JNIEXPORT jlong JNICALL Java_org_visp_mbt_VpMbGenericTracker_VpMbGenericTracker_12 (JNIEnv*, jclass, jintArray);\n",
          "JNIEXPORT jlong JNICALL Java_org_visp_mbt_VpMbGenericTracker_VpMbGenericTracker_12",
          "  (JNIEnv* env, jclass , jintArray trackerTypes)",
          "{",
          "  static const char method_name[] = \"mbt::VpMbGenericTracker_12()\";",
          "  try {",
          "   LOGD(\"%s\", method_name);",
          "   vpMbGenericTracker* _retval_ = new vpMbGenericTracker( List_to_vector_int(env, trackerTypes) );",
          "   return (jlong) _retval_;",
          "  } catch(const std::exception &e) {",
          "   throwJavaException(env, &e, method_name);",
          "  } catch (...) {",
          "   throwJavaException(env, 0, method_name);",
          "  }",
          "  return 0;",
          "}"
        ]
      }
```